build.sbt

```diff
@@ -278,6 +278,7 @@ lazy val node = (project in file("node"))
     ),
     buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion, git.gitHeadCommit),
     buildInfoPackage := "coop.rchain.node",
+    mainClass in Compile := Some("coop.rchain.node.Main"),
```

Comment:
why this is required?

Reply:
sbt can detect `Main` function automatically.  If there are multiple main in `coop.rchain.node` , sbt doesn't know which one belongs to the name `rnode`,  `mainClass in Compile := Some("coop.rchain.node.Main")` is for explicitly specifying the `Main` function for `rnode`.

If we don't put `mainClass in Compile := Some("coop.rchain.node.Main")`, there would be two binary named `block-store-migrate` and `main`

copied pasted from https://discord.com/channels/@me/702142029953761371/816649478705512448